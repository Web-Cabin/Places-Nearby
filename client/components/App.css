body {
    height: 100%;
    margin:0;
    padding:0;
  }

  h1 {
    font-family: Montserrat, sans-serif;
    padding: 42px;
    width: 1031px;
    color: rgb(72, 72, 72);
    font-size: 1.5em;
  }

<<<<<<< HEAD
  div.item {
    vertical-align: top;
    display: inline-block;
    /* To horizontally center images and caption */
    text-align: left;
    width: 350px; 
  }

=======
>>>>>>> f6d3f83e24a384281fb44343328d0ed79b5a36f3
  .propertyType {
    font-family: Nunito Sans, sans-serff; 
    padding-top: 8px;
    padding-bottom: 2px;
    font-style: bold;
    font-size: 0.75em;
    color:  rgb(72, 72, 72);
    font-weight: 700px;
    text-transform: uppercase;
  }

  .price {
    font-family: Montserrat, sans-serff; 
    padding-top: 3px;
    padding-bottom: 1px;
    font-size: .9em;
    font-weight: 400px;
    color: rgb(72,72,72);
  }

<<<<<<< HEAD
  #wrapper {
    display: flex;
  }

  #stars {
    color: rgb(0,139,139)
  }

  #reviews {
    flex: 1;
=======
  .reviews {
>>>>>>> f6d3f83e24a384281fb44343328d0ed79b5a36f3
    font-family: Montserrat, sans-serff; 
    padding-top: 3px;
    padding-bottom: 1px;
    font-size: .9em;
    font-weight: 400px;
    color: rgb(72,72,72);
  }

  .description {
    font-family: Nunito Sans, sans-serff; 
<<<<<<< HEAD
    font-size: 1.05em;
    font-weight: 600px;
    font-style: bold; 
=======
>>>>>>> f6d3f83e24a384281fb44343328d0ed79b5a36f3
    color: rgb(72, 72, 72);
  }

  .slideshow {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content: center;
    align-content: center;
    height: 100%;
    padding: 2em;
    background-color: #ffffff;
    margin: 2em auto;
    border-radius: 10px;
  }
  
  .arrow-left {
    align-self: center;
    height:0;
    width: 2em;
    padding:0;
    margin: 0.5em;
    border-top: 3em solid transparent;
    border-right: 3em solid;
    border-bottom: 3em solid transparent;
    cursor: pointer;
  }
  
  .arrow-right {
    align-self: center;
    height:0;
    width: 2em;
    padding:0;
    margin: 0.5em;
    border-top: 3em solid transparent;
    border-left: 3em solid;
    border-bottom: 3em solid transparent;
    cursor: pointer;
  }
  
  .faded-left {
    border-right: 3em solid;
  }
  
  .faded-right {
    border-left: 3em solid;
  }
  
  .image {
    object-fit: cover;
    margin: auto 0.1em;
    border-radius: 5px;
    animation: appear 1.5s ease-in;
  }
  