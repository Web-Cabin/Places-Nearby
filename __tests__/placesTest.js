import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import App from '../client/components/app.jsx';

import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('Place Component', () => {
  const testPlaceList = [
    {
      "_id" : "5d643a63646c8a700c8e1a54", 
      "id" : 28, 
      "photoUrl" : "https://nearby-recommendations.s3-us-west-1.amazonaws.com/28.jpg", 
      "title" : "Iusto veniam sed laborum ab corporis esse.", 
      "city" : "Mertzstad", 
      "propertyType" : "et reiciendis sed", 
      "price" : 169, 
      "reviews" : 119,
       "__v" : 0 
    },
    { 
      "_id" : "5d643a63646c8a700c8e1a50", 
      "id" : 24, 
      "photoUrl" : "https://nearby-recommendations.s3-us-west-1.amazonaws.com/24.jpg", 
      "title" : "Doloribus quidem consectetur animi.", 
      "city" : "Raulburgh", 
      "propertyType" : "velit quis beatae", 
      "price" : 238, 
      "reviews" : 273, 
      "__v" : 0 
    },
    { 
      "_id" : "5d643a63646c8a700c8e1a45", 
      "id" : 13,
      "photoUrl" : "https://nearby-recommendations.s3-us-west-1.amazonaws.com/13.jpg", 
      "title" : "Dicta saepe quae sit minus quae culpa est.", 
      "city" : "Kaceymouth", 
      "propertyType" : "ut enim asperiores", 
      "price" : 238, 
      "reviews" : 246, 
      "__v" : 0 
    }
  ];
  const wrapper = shallow(
    <App places={testPlaceList} />,
  );
  
  it('should render 3 places', () => {
    expect(wrapper.children().length).toBe(3);
  });

  it('should render the Place component', () => {
    expect(wrapper.find('Place').exists());
  });

  it('should have correct initial states', () => {
    expect(wrapper.state()).toEqual({ currentIndex: 0, places: [], translateValue: 0});
  });

  it('should change index on click', () => {
    wrapper.find('fadedright').simulate('click');
    expect(wrapper.state('finish')).toEqual(4);
    wrapper.find('fadedleft').simulate('click');
    expect(wrapper.state('finish')).toEqual(3);
  });

});
