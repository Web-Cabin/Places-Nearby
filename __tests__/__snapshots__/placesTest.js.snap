// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Place Component renders 1`] = `ShallowWrapper {}`;
